### ISSUE TITLE
Please, update TextBox fluent design specification.

### ISSUE THREAD

**Issue by MairwunNx** [JFXtras/jfxtras-styles#22]
![capture20181219201159957](https://user-images.githubusercontent.com/39625750/50237674-8caff980-03ce-11e9-8706-fd7232ca5396.png)

Dark theme not correctly. I think, dark colors for you textbox not correct. 

I improve dark theme for textbox, please review it, it looks better and looks like textbox in fluent.

JMetroBase.css: (left green line - changed line)

![image](https://user-images.githubusercontent.com/39625750/50237810-dc8ec080-03ce-11e9-9fac-9beba7d7a0c3.png)

JMetroDarkTheme.css: (left cyan line - changed line)

![image](https://user-images.githubusercontent.com/39625750/50237890-13fd6d00-03cf-11e9-85d1-cef0f0589535.png)

Result: (Left - Edited css, Right - Original fluent design)

![image](https://user-images.githubusercontent.com/39625750/50237922-2b3c5a80-03cf-11e9-83d0-a41e36d05718.png)

![image](https://user-images.githubusercontent.com/39625750/50237977-4dce7380-03cf-11e9-8d4f-a17f3902d8f3.png)

![image](https://user-images.githubusercontent.com/39625750/50238047-76ef0400-03cf-11e9-8508-8e801d980733.png)

![image](https://user-images.githubusercontent.com/39625750/50238309-1ad8af80-03d0-11e9-94c5-0c9504960ddb.png)

**Comment by MairwunNx** [comment]
@dukke ! Please review the issue!

**Comment by dukke** [comment]
Sorry my daily paied job has been keeping me ocupied and with no spare time for my open source projects. Is is now fixed.

**Comment by MairwunNx** [comment]
No problem @dukke , i think what you break notify, and i make mention. :)) I'm fine what you accepted changes.

**Comment by dukke** [comment]
I knew I wasn't following Fluent Design. I did it because I didn't like the hard transition when the control gets focused.
I do tend to agree with you that this looks better with your suggestion. So I changed the code. Thanks.
I did some changes to your suggestion:
- text_color variable is now the color of text when the text isn't focused
- text_focused_color variable is the text color when the control is focused
- Changed the style of text inputs by overrriding the two vars above. This way all the style is defined by overriding color vars defined in JMetroBase.css
- Changed your black color (#000) to #111. We don't want to use pure black colors.

Just one thing I'd like to add: We're not trying to be an exact copy of Fluent Design (JMetro is inspired by FD and not a direct copy) and whenever we see the chance of improving on FD, we'll do it.

Finally, thanks for this issue report. I like the high  level of detail you put into your issue reports and it helps greatly.
Thanks Pavel 👍
